app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/AboutFragment.kt

```diff
@@ -142,6 +148,24 @@ class AboutFragment : Fragment() {
                 "https://github.com/splitwise/TokenAutoComplete/",
                 "Apache License, Version 2.0",
             ),
+            Library("accompanist","https://github.com/google/accompanist","Apache License, Version 2.0"),
```

Comment:
The documentation capitalizes the first letter of the library name: Accompanist. We should do the same.